### PR TITLE
docs: update shared mem. info in GUCS max_fsm_pages and max_connections

### PR DESCRIPTION
**Pull request by mkiyama**
Add info from support about shared memory conflicts.

PR for 5X_STABLE
Will be ported to MAIN - NOTE: max_fsm* GUCs removed in main.

### CODE REVIEW

**Comment by hlinnaka** [comment]
Note that in 'master', the max_fsm_* settings have been removed, as part of the 8.4 merge.

**Comment by mkiyama** [comment]
@hlinnaka - thanks for pointing that out.

File: gpdb-doc/dita/ref_guide/config_params/guc-list.xml [hunk 1]
```diff
@@ -6994,10 +6994,13 @@
         parameter, <xref href="#max_prepared_transactions" format="dita"/> must be increased as
         well. For more information about limiting concurrent connections, see "Configuring Client
         Authentication" in the <i>Greenplum Database Administrator Guide</i>.</p>
-      <p>Increasing this parameter may cause Greenplum Database to request more shared
-        memory.Increasing this parameter might cause Greenplum Database to request more shared
-        memory. See <xref href="#shared_buffers" format="dita"/> for information about Greenplum
-        server instance shared memory buffers.</p>
+      <p>Increasing this parameter may cause Greenplum Database to request more shared memory.
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/config_params/guc-list.xml]: First sentence is repeated here (slight variation).

File: gpdb-doc/dita/ref_guide/config_params/guc-list.xml [hunk 2]
```diff
@@ -7055,7 +7058,10 @@
     <title>max_fsm_pages</title>
     <body>
       <p>Sets the maximum number of disk pages for which free space will be tracked in the shared
-        free-space map. Six bytes of shared memory are consumed for each page slot. </p>
+        free-space map. Six bytes of shared memory are consumed for each page slot. If this value is
+        increased, <xref href="#shared_buffers" format="dita"/> and the kernel parameter
+          <codeph>SHMMAX</codeph> might also need to be increased. Also, parameters that depend
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/config_params/guc-list.xml]: "depend" -> "depend on"

File: gpdb-doc/dita/ref_guide/config_params/guc-list.xml [hunk 3]
```diff
@@ -6994,10 +6994,13 @@
...
-      <p>Increasing this parameter may cause Greenplum Database to request more shared
-        memory.Increasing this parameter might cause Greenplum Database to request more shared
-        memory. See <xref href="#shared_buffers" format="dita"/> for information about Greenplum
-        server instance shared memory buffers.</p>
+      <p>Increasing this parameter may cause Greenplum Database to request more shared memory.
+        Increasing this parameter might cause Greenplum Database to request more shared memory. See
+          <xref href="#shared_buffers" format="dita"/> for information about Greenplum server
+        instance shared memory buffers. If this value is increased, <xref href="#shared_buffers"
+          format="dita"/> and the kernel parameter <codeph>SHMMAX</codeph> might also need to be
+        increased. Also, parameters that depend shared memory such as <xref href="#max_fsm_pages"/>
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/config_params/guc-list.xml]: "depend" -> "depend on"

**Comment by mkiyama** [comment]
committed updates based on review comments.

**Pull request merged**